MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Ascension/Effect/AscensionEffectBuilderImpl.java

```diff
@@ -11,24 +11,29 @@
 
 //TODO look at java patterns and figure out a nice way to not have to repeat the builder multiple times
 public abstract class AscensionEffectBuilderImpl<E extends AscensionEffect> implements AscensionEffectBuilder {
-    private long offset = 30;
+    private int offset = 0;
```

Comment:
Any reason for changing to int?

Also why is the default 0?

Reply:
So i won't have to convert it to int when it's used with the schedule system or other functions.
It's in 0 by default since this is an optional function, as such it won't have unexpected outcomes when not specified.

Reply:
Ok, sounds good